src/main/resources/static/js/projects/submissions-controller.js

```diff
@@ -0,0 +1,53 @@
+angular.module('CapstoneSystem')
+    .controller('SubmissionsController', function ($scope, User, Project) {
+        var table = $('#submissionsTable').DataTable( {

```

Comment:
There is no need to initialize the datatable this way anymore. You can just add "datatable" to the html table element and it automatically does it for you 😄

Reply:
Cool, didn’t know that 👍🏻